src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -457,28 +457,66 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
+            emails.add(generateFeedbackSessionClosedEmailForStudents(course, session, student.name, student.email));
         }
 
         return emails;
     }
-
-    private EmailWrapper generateFeedbackSessionClosedEmail(CourseAttributes course,
+    
+    /**
+     * Generates the feedback session closed email for the students with the given {@code userName} and {@code userEmail}.
+     */
+    private EmailWrapper generateFeedbackSessionClosedEmailForStudents(CourseAttributes course,
             FeedbackSessionAttributes session, String userName, String userEmail) {
-        String template = EmailTemplates.USER_FEEDBACK_SESSION_CLOSED;
+    	
+    	String template = EmailTemplates.USER_FEEDBACK_SESSION_CLOSED;
+    	
         String subject = EmailType.FEEDBACK_CLOSED.getSubject();
-
+        
         String emailBody = Templates.populateTemplate(template,
-                "${userName}", SanitizationHelper.sanitizeForHtml(userName),
+        		"${userName}", SanitizationHelper.sanitizeForHtml(userName),
+        		"${instructorFragment}", "",
                 "${courseName}", SanitizationHelper.sanitizeForHtml(course.getName()),
                 "${courseId}", SanitizationHelper.sanitizeForHtml(course.getId()),
                 "${feedbackSessionName}", SanitizationHelper.sanitizeForHtml(session.getFeedbackSessionName()),
                 "${deadline}", SanitizationHelper.sanitizeForHtml(TimeHelper.formatTime12H(session.getEndTime())),
                 "${supportEmail}", Config.SUPPORT_EMAIL);
-
+        
+        EmailWrapper email = getEmptyEmailAddressedToEmail(userEmail);
+        email.setSubject(String.format(subject, course.getName(), session.getFeedbackSessionName()));
+        email.setContent(emailBody);
+        return email;
+    }
+    
+    /**
+     * Generates the feedback session closed email for the instructors with the given {@code userName} and {@code userEmail}.
+     */
+    private EmailWrapper generateFeedbackSessionClosedEmailForInstructors(CourseAttributes course,
+            FeedbackSessionAttributes session, String userName, String userEmail) {
+    	
+    	String template = EmailTemplates.USER_FEEDBACK_SESSION_CLOSED;
+    	
+        String subject = EmailType.FEEDBACK_CLOSED.getSubject();
+        
+        String emailBody = Templates.populateTemplate(template,
+        		"${userName}", SanitizationHelper.sanitizeForHtml(userName),
+        		"${instructorFragment}",
+        			    "<p>The email below has been sent to students of course: "
+        			    + SanitizationHelper.sanitizeForHtml(course.getId()) + ", " 
+        			    + SanitizationHelper.sanitizeForHtml(course.getName())
+        			    + ".<br>" + Const.EOL + "<br>" + Const.EOL
+        			    + "=== Email message as seen by the students ===</p>" + Const.EOL,
+                "${courseName}", SanitizationHelper.sanitizeForHtml(course.getName()),
+                "${courseId}", SanitizationHelper.sanitizeForHtml(course.getId()),
```

Comment:
This is because, in the case of the instructor's email, we would need the course's details both on the preamble and in the body of the email (the one that is shown to students). Thus, I thought that we would need the replacement of the name and the ID twice. Is this wrong?